src/widget/wtrackmenu.cpp

```diff
@@ -2422,47 +2424,56 @@ void WTrackMenu::slotRemoveFromDisk() {
         // Purge only those tracks whose files have actually been deleted.
         m_pLibrary->trackCollectionManager()->purgeTracks(tracksToPurge);
 
-        // Show purge summary message
-        QMessageBox msgBoxPurgeTracks;
-        msgBoxPurgeTracks.setIcon(QMessageBox::Information);
-        QString msgTitle;
-        QString msgText;
-        if (m_pTrackModel) {
+        if (s_showPurgesuccessPopup) {
+            // Show purge summary message
+            QMessageBox msgBoxPurgeTracks;
+            msgBoxPurgeTracks.setIcon(QMessageBox::Information);
+            QString msgTitle;
+            QString msgText;
+            if (m_pTrackModel) {
 #if QT_VERSION >= QT_VERSION_CHECK(5, 15, 0)
-            msgTitle = tr("Track Files Deleted");
+                msgTitle = tr("Track Files Deleted");
 #else
-            msgTitle = tr("Track Files Moved To Trash");
+                msgTitle = tr("Track Files Moved To Trash");
 #endif
```

Comment:
The `#if` for Qt5 can be removed, now that we have fully migrated to Qt6 (at least to my knowledge, correct me if I'm wrong)

```suggestion
                msgTitle = tr("Track Files Moved To Trash");
```

Also, the `#if` condition was already wrong/exactly backwards - compare with:
https://github.com/mixxxdj/mixxx/blob/edae0ca3b2fc36c51d9f310f341fb2e4bbea7d26/src/widget/wtrackmenu.cpp#L341-L345